src/components/views/TrackView.js

```diff
@@ -1,40 +1,49 @@
 import React, {Component} from "react";
 import {Helmet} from "react-helmet";
-import {Timeline, TimelineEvent} from 'react-event-timeline';
 import SearchResult from "../search/SearchResult";
 import TimeLineEndBlock from "../common/TimeLineEndBlock";
 import RiderSearchContainer from '../../containers/RiderSearchContainer'
+import TrackTimeline from "../common/TrackTimeline";
 
 class TrackView extends Component {
     render() {
       const {judgments} = this.props;
       const {search} = this.props;
       const {raceGroups} = this.props;
+      const {stage} = this.props;
 
-        const divStyle = {
-            margin: '0 0 3rem 0'
-        };
 
-        const iconStyle = {
-            backgroundColor: '#6fba1c'
-        };
+      let elements = [];
+      judgments.map(judgment => {elements.push({distance : judgment.distance, text : judgment.name})});
+      raceGroups.map(raceGroup => {elements.push({distance : 90, text : raceGroup.raceGroupType})});
+      elements.sort((a,b) => b.distance - a.distance);
+      let elems = [];
+      let lastElement = null;
+      let size = elements.length;
+      let sizeBottom = 0;
+      elements.map((elem, i) => {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add a "return" statement to this callback. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3796)